mlrun/model.py

```diff
@@ -378,6 +420,16 @@ def inputs(self):
     def inputs(self, inputs):
         self._inputs = self._verify_dict(inputs, "inputs")
 
+    @property
+    def hyper_options(self) -> HyperParamOptions:
+        return self._hyper_options
+
+    @hyper_options.setter
+    def hyper_options(self, hyper_options):
+        self._hyper_options = self._verify_dict(
+            hyper_options, "inputs", HyperParamOptions
```

Comment:
Change the name, `inputs` is where you copy-pasted from

Reply:
done